tests/unitTests/api_client/client.test.ts

```
@@ -1,18 +1,72 @@
+        };
+    }
+
+    public async ping(): Promise<GenericResponse<string>> {
+        return await this.pingClient();
+    }
+
+    public async chainInfo(): Promise<GenericResponse<ChainInfo>> {
+        return await this.chainInfoClient();
+    }
+
+    public async selectQuery(query: SelectQueryRequest): Promise<GenericResponse<Object[]>> {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)